src/us/kbase/workspace/test/database/provenance/CreditMetadataTest.java

```diff
@@ -0,0 +1,1049 @@
+package us.kbase.workspace.test.database.provenance;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import us.kbase.common.test.TestCommon;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Collections;
+import java.util.Optional;
+
+import static us.kbase.common.test.TestCommon.opt;
```

Comment:
I always put all static imports first just because that's how I've seen it done *shrug*

Reply:
I'll move them in the next PR.